.github/actions/tag-pruner/README.md

```diff
@@ -0,0 +1,39 @@
+# Sofie Tag Pruner
+
+Use this action to clean up tags in GitHub repositories.
+
+## Usage
+
+```yaml
+uses: nrkno/sofie-github-workflows/.github/actions/tag-pruner@v1
+with:
+  # Tags older than the specified number of days will be deleted (required)
+  older-than: 30
+  
+  # Regex pattern for tags that should be ignored
+  ignore: v[0-9]+\.[0-9]+\.[0-9]+.*
```

Comment:
If you have several patterns that should be ignored, how should they be input?

Reply:
Currently by writing a very complex regex pattern that covers all of them?
Not sure what happens if you try to pass an array as an input to an action, the typings suggest all inputs are converted to string but I'll try it and see what happens!

Reply:
So it literally takes the value of an environment variable called `INPUT_IGNORE` and passes that back as a string: https://github.com/actions/toolkit/blob/03d6c2479c60c5873d1959fe1f167cc57dccccfc/packages/core/src/core.ts#L126

I guess realistically the options are either to have multiple regex input options, E.G. ignore-1, ignore-2 etc.
Or we could say that a comma is used as a delimiter for multiple patterns so you could write something like: `pattern1,pattern2` although that feels a little messy and means we can't have commas in the regex? (Probably not a big problem)
Or to write a complex pattern?

Reply:
I guess one complex pattern works, I just like regex to be as simple as possible.